src/api/spec/features/beta/webui/attributes_spec.rb

```diff
@@ -73,6 +73,7 @@
           login user
 
           visit index_attribs_path(project: user.home_project_name)
+          first('table tbody tr td').click if mobile?
```

Comment:
You should tell us why in the commit body...

Reply:
Added a comment in the commit body.